epochStart/metachain/systemSCs.go

```diff
@@ -572,6 +583,34 @@ func (s *systemSCProcessor) updateSystemSCConfigMinNodes() error {
 	return err
 }
 
+func (s *systemSCProcessor) resetLastUnJailed() error {
+	vmInput := &vmcommon.ContractCallInput{
+		VMInput: vmcommon.VMInput{
+			CallerAddr: s.endOfEpochCallerAddress,
+			Arguments:  [][]byte{},
+			CallValue:  big.NewInt(0),
+		},
+		RecipientAddr: s.stakingSCAddress,
+		Function:      "resetLastUnJailedFromQueue",
+	}
+
+	vmOutput, err := s.systemVM.RunSmartContractCall(vmInput)
+	if err != nil {
+		return err
+	}
+
+	if vmOutput.ReturnCode != vmcommon.Ok {
+		return epochStart.ErrResetLastUnJailedFromQueue
+	}
+
+	err = s.processSCOutputAccounts(vmOutput)
```

Comment:
return s.processSCOutputAccounts(vmOutput) directly

Reply:
would leave like this.